editor/editor_file_system.cpp

```diff
@@ -2701,6 +2701,7 @@ EditorFileSystem::EditorFileSystem() {
 	using_fat32_or_exfat = (da->get_filesystem_type() == "FAT32" || da->get_filesystem_type() == "exFAT");
 
 	scan_total = 0;
+	scanning_changes_done.set(false);
```

Comment:
Isn't that the default value already?

Reply:
It is -- I missed the default value getting set in the constructor when I was reading the template.